pkg/multitenant/mtinfopb/info.proto

```diff
@@ -83,7 +83,12 @@ message ProtoInfo {
   // VIRTUAL CLUSTER FROM REPLICATION STREAM.
   optional util.hlc.Timestamp last_revert_tenant_timestamp = 8 [(gogoproto.nullable) = false];
 
-  // Next ID: 9
+  // ReadFromTeanant is the ID, if any, of another tenant from which this tenant
+  // should be configured to serve read-only SQL requests (by configuring and
+  // maintaining its catalog accordingly).
+  optional uint64 read_from_tenant = 9  [(gogoproto.nullable) = false];
```

Comment:
use a roachpb.TenantID instead?

Reply:
done

Reply:
Ah, I'd forgotten why I did this but the test suite has reminded me: the gogoproto zero value handling! If we use TenantID instead of the uint64, we change the marshaled zero value tenant record, which breaks some tests.